torch/lib/nccl/src/common_coll.h

```diff
@@ -17,7 +17,7 @@ static ncclResult_t PointerCheck(const void* pointer, struct ncclComm* comm, con
     return ncclInvalidDevicePointer;
   }
   if (attr.memoryType == cudaMemoryTypeDevice && attr.device != comm->cudaDev) {
-    WARN("%s : %s allocated on device %d mismatchs with NCCL device %d \n", opname, ptrname, attr.device, comm->cudaDev);
+    WARN("%s : %s allocated on device %d mismatches with NCCL device %d \n", opname, ptrname, attr.device, comm->cudaDev);
```

Comment:
I missed this one as well. Could you remove this fix in `nccl` to simplify the merge process?
Actually, @soumith is that right? Is it easy to keep both torch and pytorch in sync with PRs in pytorch touching several repos in torch?